assets/js/blocks/price-filter/frontend.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { render } from 'react-dom';
+
+/**
+ * Internal dependencies
+ */
+import Block from './block.js';
+
+const containers = document.querySelectorAll(
+	'.wp-block-woocommerce-price-filter'
+);
+
+if ( containers.length ) {
+	Array.prototype.forEach.call( containers, ( el ) => {
```

Comment:
You could just do `containers.forEach` here?

Reply:
This is needed for ie11 compatibility, since we're not using lodash helpers.

Reply:
Existing pattern in existing code FYI